deployments/gpu-operator/templates/clusterrole.yaml

```diff
@@ -102,8 +102,10 @@ rules:
   - nvidia.com
   resources:
   - clusterpolicies
+  - clusterpolicies/finalizers
```

Comment:
adding these perms to the gpu-operator clusterroles, as it needs to have the permissions in order to grant the same permissions  to another entity